examples/quickstart-lerobot/pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.metadata]
+allow-direct-references = true
+
+[project]
+name = "lerobot_example"
+version = "1.0.0"
+description = "Federated Learning with Hugginface LeRobot transformers and Flower (Quickstart Example)"
+license = "Apache-2.0"
+authors = [
+    { name = "The Flower Authors", email = "hello@flower.ai" },
+    { name = "Kaushik Amar Das", email = "kaushik.das@iiitg.ac.in" },
+]
+
+dependencies = [
+    "flwr[simulation]==1.12.0",
```

Comment:
```suggestion
    "flwr[simulation]==1.13.1",
```

Reply:
Let's set as minimum the most recent version of Flower. Also, it seems we need to include the `transformers` package.